src/signer/ledger.ts

```diff
@@ -0,0 +1,63 @@
+import { hexZeroPad } from '@ethersproject/bytes';
+import Eth from '@ledgerhq/hw-app-eth';
+import Transport from '@ledgerhq/hw-transport';
+import TransportWebHID from '@ledgerhq/hw-transport-webhid';
+import { ethers } from 'ethers';
+
+import { Abi, Invocation, InvocationsSignerDetails, Signature } from '../types';
+import { TypedData } from '../utils/typedData';
+import { SignerInterface } from './interface';
+
+export class LedgerSigner implements SignerInterface {
+  public derivationPath = "/2645'/579218131'/1148870696'/0'/0'/0";
+
+  static transport: Transport;
```

Comment:
```suggestion
  static transport: Transport | undefined;
```

and should this be private?